image/v3/modifying/text-fonts.md

```diff
@@ -49,21 +49,21 @@ $image = ImageManager::imagick()->read('images/example.jpg');
 
 // write text to image
 $image->text('The quick brown fox', 120, 100, function ($font) {
-    $font->filename('./fonts/comic-sans.ttf');
-    $font->color('#b01735');
-    $font->size(70);
-    $font->align('center');
-    $font->valign('middle');
-    $font->lineHeight(1.6);
-    $font->angle(10);
+    $font->setFilename('./fonts/comic-sans.ttf');
+    $font->setColor('#b01735');
+    $font->setSize(70);
+    $font->setAlignment('center');
+    $font->setValignment('middle');
+    $font->setLineHeight(1.6);
+    $font->setAngle(10);
 });
 ```
 
 ### Font size
 
-> public FontInterface::size(float $size): FontInterface
+> public FontInterface::setSize(float $size): FontInterface
```

Comment:
Please revert. This is correct call of class Intervention\Image\Typography\FontFactory::class.